astroid/nodes/as_string.py

```diff
@@ -449,7 +481,10 @@ def visit_return(self, node):
         """return an astroid.Return node as string"""
         if node.is_tuple_return() and len(node.value.elts) > 1:
             elts = [child.accept(self) for child in node.value.elts]
-            return f"return {', '.join(elts)}"
+            tmp = f"{', '.join(elts)}"
+            tmp = f"return ({tmp})"
+            return tmp
+            # return f"return {', '.join(elts)}"
```

Comment:
Same here, why?

Reply:
I looked for the previous test, which was about the third-party library "Sklearn".
The code segment is about the modification of visit_return node.

For example, 
you can test the file --> Sklearn/discriminant_analysis.py/

def _decision_function(self, X):
        .....
&nbsp;&nbsp;&nbsp;&nbsp;return (-0.5 * (norm2 + u) + np.log(self.priors_))


as_string.py will transfer it as "return -0.5 * (norm2 + u) + np.log(self.priors_)" with missing the parentheses.

Reply:
Even without the parentheses the result is valid. The goal of the `AsStringVisitor` isn't to create the exact input code, just a valid python code string. Some information just gets lost when the code is parsed to the AST. (There are libraries that handle this better though.)

Could you elaborate a bit more on the context you encountered the errors? I understand you mentioned `Sklearn`, but what were you trying to accomplish? Run pylint? A custom plugin? Something else entirely?